README.md

```diff
@@ -15,7 +15,19 @@ npm install gavel2html
 ```javascript
 Gavel2Html = require('gavel2html');
 
-g2h = Gavel2Html.new(realData, expectedData, gavelOutput);
+g2h = Gavel2Html.new({
```

Comment:
Should this be `g2h = new Gavel2Html({ ... });`?

The older did not work either, right?